Update documentation for metrics data retention (#153)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_preparing-your-openshift-environment-for-stf.adoc

* For information about sizing requirements for ElasticSearch, see https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-managing-compute-resources.html.

[id="metrics-retention-time-period"]
== Metrics retention time period

> Metrics are stored in Prometheus for a time period of 24 hours by default, providing an overview of the cloud systems and enough data to allow for trends to develop for alerting. To adjust STF for additional metrics retention time set a new value in `backends.metrics.prometheus.storage.retention` such as `7d` for seven days. Long retention periods are not recommended as returning data from heavily populated Prometheus systems could result in queries returning slowly.

Reviewer: ```suggestion
By default, metrics are stored in Prometheus for a time period of 24 hours, providing an overview of the cloud systems and enough data to allow for trends to develop for alerting. To adjust {ProjectShort} for additional metrics retention time, set a new value in `backends.metrics.prometheus.storage.retention`, for examplle, `7d` for seven days. If you use long retention periods, returning data from heavily populated Prometheus systems can result in queries returning slowly.
```

The first sentence needs to be adjusted but I've been looking at it for ages and drawn a blank. I'll come back to this another time because it's not a priority.

Author: How about something like....

The default retention time for metrics stored in {ProjectShort} is 24 hours which provides enough data to allow for trends to develop for the purposes of alerting.

> By default, metrics are stored in Prometheus for a time period of 24 hours, providing an overview of the cloud systems and enough data to allow for trends to develop for alerting. To adjust {ProjectShort} for additional metrics retention time, set a new value in `backends.metrics.prometheus.storage.retention`, for example, `7d` for seven days. If you use long retention periods, returning data from heavily populated Prometheus systems can result in queries returning slowly.

Reviewer: ```suggestion
The default retention time for metrics stored in {ProjectShort} is 24 hours, which provides enough data to allow for trends to develop for the purposes of alerting. To adjust {ProjectShort} for additional metrics retention time, set a new value in `backends.metrics.prometheus.storage.retention`, for example, `7d` for seven days. If you use long retention periods, returning data from heavily populated Prometheus systems can result in queries returning slowly.
```

> For long term storage, use of systems designed for long term data retention such as https://thanos.io/[Thanos], should be used.

Reviewer: ```suggestion
For long-term storage, use systems designed for long-term data retention, for example, https://thanos.io/[Thanos].
```
